### PR TITLE
Uniformization of envvars and prerun in Lsf and Slurm

### LINKED ISSUE #525: Conda environment for queue jobs
When *Psi4* jobs are queue via Slurm (or the other queues, except the local), *conda* environment activation is lost.

This is problem, if you want to switch to a development version of *Psi4* on a specific *conda* environment.

We need a more general solution to this hack:
```diff
diff --git a/htmd/queues/slurmqueue.py b/htmd/queues/slurmqueue.py
index 009dcb5..71d9f64 100644
--- a/htmd/queues/slurmqueue.py
+++ b/htmd/queues/slurmqueue.py
@@ -141,6 +141,9 @@ class SlurmQueue(SimQueue, ProtocolInterface):
             # Trap kill signals to create sentinel file
             f.write('\ntrap "touch {}" EXIT SIGTERM\n'.format(os.path.normpath(os.path.join(workdir, self._sentinel))))
             f.write('\ncd {}\n'.format(workdir))
+
+            f.write('. /shared/raimis/opt/miniconda3/bin/activate htmd_opt\n')
+
             f.write('{}'.format(runsh))
 
             # Move completed trajectories
```

### PR DESCRIPTION
**Pull request by j3mdamas**
As per comment: https://github.com/Acellera/htmd/issues/525#issuecomment-369069037

It will close #525

### CODE REVIEW

**Comment by j3mdamas** [comment]
Wait, I forgot something on one of them. Going to commit now.

**Comment by j3mdamas** [comment]
This is also why I put tests as a priority to solve in issue #575, as these small errors are not tested (and object instatiation tests should be enough to catch them).

**Comment by j3mdamas** [comment]
Great, it already passed Travis.

File: htmd/queues/lsfqueue.py [hunk 1]
```diff
@@ -68,14 +68,19 @@ def __init__(self, _configapp=None):
         self._arg('ncpu', 'int', 'Number of CPUs to use for a single job', self._defaults['ncpu'],
                   val.Number(int, '0POS'))
         self._arg('memory', 'int', 'Amount of memory per job (MB)', self._defaults['memory'], val.Number(int, '0POS'))
-        self._arg('walltime', 'int', 'Job timeout (hour:min or min)', self._defaults['walltime'], val.Number(int, '0POS'))
+        self._arg('walltime', 'int', 'Job timeout (hour:min or min)', self._defaults['walltime'],
+                  val.Number(int, '0POS'))
         self._arg('resources', 'list', 'Resources of the queue', self._defaults['resources'], val.String(), nargs='*')
-        self._arg('environment', 'list', 'Things to run before the job (sourcing envs).', self._defaults['environment'],
-                  val.String(), nargs='*')
+        self._cmdDeprecated('environment', 'prerun')
```
Comment (stefdoerr) [inline htmd/queues/lsfqueue.py]: Shouldn't it be `'environment', 'envvars'`?

Comment (j3mdamas) [reply htmd/queues/lsfqueue.py]: In LsfQueue, the `environment` meant commands to run before running the application (while in Slurm meant passing environment variables). That was the inconsistency and why a deprecation of the name was needed, and two new names were needed.

Comment (stefdoerr) [reply htmd/queues/lsfqueue.py]: Ok great!

**Pull request merged**